src/api/entities/Asset/index.ts

```diff
@@ -185,10 +183,9 @@ export class Asset extends Entity<UniqueIdentifiers, string> {
    * Transfer ownership of the Asset to another Identity. This generates an authorization request that must be accepted
    *   by the recipient
    *
-   * @note this will create [[AuthorizationRequest | Authorization Requests]] which have to be accepted by
-   *   the corresponding [[Account | Accounts]] and/or [[Identity | Identities]].
-   *   An Account or Identity can fetch its pending Authorization Requests by calling {@link Authorizations.getReceived}.
-   *   Also, an Account or Identity can directly fetch the details of an Authorization Request by calling {@link Authorizations.getOne}
+   * @note this will create {@link AuthorizationRequest Authorization Request} which has to be accepted by the `target` Identity.
```

Comment:
```suggestion
   * @note this will create {@link AuthorizationRequest | Authorization Request} which has to be accepted by the `target` Identity.
```